src/datasets/config.py

```diff
@@ -183,7 +183,9 @@
 
 # Batch size constants. For more info, see:
 # https://github.com/apache/arrow/blob/master/docs/source/cpp/arrays.rst#size-limitations-and-recommendations)
-DEFAULT_MAX_BATCH_SIZE = 1000
+DEFAULT_MAX_BATCH_SIZE = 1024 * 1024
```

Comment:
This is the default arrow row group size. If we choose a small row group size then we cannot profit from CDC chunking that much.